src/components/ClusterNodes/ClusterSelectionList.tsx

```diff
@@ -121,11 +72,15 @@ const ClusterSelectionList: React.FC<ClusterSelectionType> = ({
         return value
     }
 
+    const formatTimeString = (start: Dayjs, now: Dayjs) => {
```

Comment:
It is a util, check if already present (check with @vivek-devtron) else create

Reply:
We should use the common one

Reply:
using the common one now. Deleted the code that extended dayjs in dashboard with the required plugin